### DISCUSSION TITLE
Error testing lammps using homebrew

### DISCUSSION THREAD

**Discussion by DynamicsTopG** [Homebrew/discussions#5010] [Getting started]
### Output of `brew config`

```shell
% brew test lammps -v
==> Installing 'bundler' gem
ERROR:  SSL verification error at depth 0: self-signed certificate (18)
ERROR:  Certificate /C=US/ST=USA/L=New York/O=Optimization/OU=Optimization/CN=tomcat is not trusted
ERROR:  SSL verification error at depth 0: self-signed certificate (18)
ERROR:  Certificate /C=US/ST=USA/L=New York/O=Optimization/OU=Optimization/CN=tomcat is not trusted
ERROR:  SSL verification error at depth 0: self-signed certificate (18)
ERROR:  Certificate /C=US/ST=USA/L=New York/O=Optimization/OU=Optimization/CN=tomcat is not trusted
Error: failed to install the 'bundler' gem.
```

### Output of `brew doctor`

```shell
% brew test lammps -v
==> Installing 'bundler' gem
ERROR:  SSL verification error at depth 0: self-signed certificate (18)
ERROR:  Certificate /C=US/ST=USA/L=New York/O=Optimization/OU=Optimization/CN=tomcat is not trusted
ERROR:  SSL verification error at depth 0: self-signed certificate (18)
ERROR:  Certificate /C=US/ST=USA/L=New York/O=Optimization/OU=Optimization/CN=tomcat is not trusted
ERROR:  SSL verification error at depth 0: self-signed certificate (18)
ERROR:  Certificate /C=US/ST=USA/L=New York/O=Optimization/OU=Optimization/CN=tomcat is not trusted
Error: failed to install the 'bundler' gem.
```

### Description of issue

Hello everyone! I have recently downloaded Homebrew and LAMMPS through it. When I try to test LAMMPS by writing in my terminal "brew test lammps -v" or "brew test lammps" I always find myself getting the same error: "Error: failed to install the 'bundler' gem."

Since I am quite new to using my laptop terminal and Homebrew I don't know what should I do next. 
Thank you so much for your help!

**Comment by SMillerDev** [comment]
What are you hoping to do with this command? Because it will only run the package tests, which generally isn't very interesting for users.

**Comment by ploxiln** [comment]
I made a custom formula, with a test, and wanted to run "brew test" on it. I'm also behind a corporate firewall that does TLS interception. I also see the same errors as reported above: SSL verification error: self-signed certificate, failed to install the 'bundler' gem.

The system curl does use the system certs which include the interception CA root, and most things work, including "brew install" etc.

But when "brew test" runs "gem" I can't seem to get it to use my own CA root bundle, even after copying it over `/opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/libexec/cert.pem`.
